external/ntpclient/ntpclient.c

```diff
@@ -89,6 +89,18 @@
 #define CONFIG_NETUTILS_NTPCLIENT_DEBUG		0
 #endif
 
+#ifndef CONFIG_NETUTILS_NTPCLIENT_DEFAULT_SERVER
+#define CONFIG_NETUTILS_NTPCLIENT_DEFAULT_SERVER		"0.ubuntu.pool.ntp.org"
+#endif
+
+#ifndef CONFIG_NETUTILS_NTPCLIENT_DEFAULT_SERVER_PORT
+#define CONFIG_NETUTILS_NTPCLIENT_DEFAULT_SERVER_PORT		123
+#endif
+
+#ifndef CONFIG_NETUTILS_NTPCLIENT_DEFAULT_INTERVAL_SECONDS
+#define CONFIG_NETUTILS_NTPCLIENT_DEFAULT_INTERVAL_SECONDS		86400
+#endif
+
```

Comment:
why didn't use config.h?

Reply:
I use .config.h. please refer below code
https://github.com/Samsung/TizenRT/pull/1222/files#diff-84e4fee7c7a2b369575f4d5f2de40e52

The role of the added code prevents the error from occurring if the following config is not set in kconfig or defconfig file.

- NETUTILS_NTPCLIENT_DEFAULT_SERVER
- NETUTILS_NTPCLIENT_DEFAULT_SERVER_PORT
- NETUTILS_NTPCLIENT_DEFAULT_INTERVAL_SECONDS